knowledge_repo/converters/ipynb.py

```diff
@@ -1,11 +1,23 @@
 import os
+import nbconvert
```

Comment:
Importing `nbconvert` here adds a hard dependency on `nbconvert`, which we would prefer to avoid. We also don't need it here, since we never using anything directly from the `nbconvert` package namespace.